### ISSUE TITLE
[Keyboard Navigation- Bot Framework Web Chat- Video]: Maximize and more option controls in video section are not accessible using keyboard

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5360]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Users who rely on the keyboard will be affected if the controls are not accessible with the keyboard. Keyboard users cannot perform the action as the control doesn't get accessible with the keyboard. Keyboard users will be affected as they cannot use this functionality.

### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Video word in type your message text box.
4. Navigate to the search results.
5. Verify the controls are accessible using keyboard or not.

### What do you expect?

Maximize and more option controls in video section should be accessible using keyboard.

### What actually happened?

Maximize and more option controls in video section are not accessible using keyboard.
### Observation:
using mouse maximize and more option controls are accessible.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/f9cd59fe-6d91-4ba7-9807-71511b3bb565)
https://github.com/user-attachments/assets/35202172-e145-4505-8855-37ca8ea5a1a8

### MAS reference

MAS 2.1.1 – Keyboard

### WCAG reference

2.1.1 – Keyboard

**Comment by manojs5** [comment]
#A11yMAS;#A11ySev1;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#MAS2.1.1;#Win11-Edge(Chromium);

**Comment by compulim** [comment]
Similar to #5370. The video control is owned by Edge as part of their `<video controls>` HTML tag.

Various control buttons can be focused by pressing <kbd>DOWN</kbd> after the video control is focused. It's deliberate choice done by Edge of choosing <kbd>DOWN</kbd> over <kbd>TAB</kbd>.

I verified the play button and various other buttons can be focused and tapped using the <kbd>DOWN</kbd> navigation pattern.
